RepositoryCompiler/CodeModel/CaDETModel/CodeItems/CaDETClass.cs

```diff
@@ -26,6 +26,9 @@ public string ContainerName
         public List<CaDETMember> Members { get; internal set; }
         public List<CaDETField> Fields { get; internal set; }
         public Dictionary<CaDETMetric, double> Metrics { get; internal set; }
+        public List<CaDETClass> FieldTypes { get; internal set; }
```

Comment:
This can be calculated fields (i.e., methods) instead of having duplicated data. E.g., GetFieldTypes returns Members.Fields.SelectMany(f => f.Type);

This saves memory and processing power, since we will only sometimes want to access a class's fieldTypes and when we do we will call this only once.

Reply:
Member nema Fields?